nodes/core/parsers/70-YAML.html

```diff
@@ -0,0 +1,47 @@
+<!--
```

Comment:
No need for copyright section anymore :-)... we can make do with one at the top level